### PR TITLE
Add autosaving to Moderator notes

### PR DESCRIPTION
**Pull request by Sonjajo**
### **What's new**
- Moderator notes autosave!! They also save whenever the drawer is closed, which occurs when you click the close button, on a card, or on the center organization spread (orgDetail).

### **Next Steps**
- Style the notes button and entire drawer
- Add Bry's toast component

### **Questions**
- Should I add a "save" button to the drawer?
- Should I add the clickAwayListener? It would be so when someone clicks on anywhere off the drawer, not just on a different card on the left, the drawer closes. I removed it this morning because I realized it wasn't being used. Tried briefly to use it but it was breaking some logic and didn't have time to figure it out. 
Instead, I added onClick={handleDrawerCloseRight} to the orgDetail and this seems to work, besides some eslint complaints. What do you think of this solution?

CC: @kwfk

### CODE REVIEW

**Comment by elixawu** [comment]
nice! imo at least, a save button isn't necessarily needed since we have the autosave. also, the clickAwayListener sounds good, but since you already have it closing on center spread, cards, or close button, i'm not sure if it's 100% needed? would be nice if it worked though

**Comment by kwfk** [comment]
> Should I add a "save" button to the drawer?

I don't think that would be necessary. However, one case that will be necessary to implement later is saving on component close. This is when a user leaves the page while the notepad is open.

> Should I add the clickAwayListener? It would be so when someone clicks on anywhere off the drawer, not just on a different card on the left, the drawer closes.

If possible, I think this would be nice to have. If this is implemented I think the logic for the drawer in the clickCard can be removed.

**Comment by kwfk** [comment]
Merging this PR into #36 because this is branched off of it

File: migrations/20201123023013-create-application_notes.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+): Promise<void> {
+  db.runSql(
+    `
+    CREATE TABLE application_notes
+      (
+        id                           SERIAL,
+        note                         TEXT,
+        created_at                   TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+        updated_at                   TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+        organization_id              INTEGER,
```
Comment (kwfk) [inline migrations/20201123023013-create-application_notes.ts]: changing this to NOT NULL because all application notes should have an organization id

**Review by kwfk — APPROVED** [review]
Really nice work!

**Pull request merged**